agent/consul/server_test.go

```diff
@@ -128,6 +130,10 @@ func testServerConfig(t *testing.T) (string, *Config) {
 	return dir, config
 }
 
+func MakeMeATestServer(t *testing.T) (string, *Server) {
```

Comment:
Did this need to be exported? I suggest if it does we should follow Go idiomatic naming:
```suggestion
func NewTestServer(t *testing.T) (string, *Server) {
```

Reply:
This appeared to be unused so I removed it.